docs/source/integrations/mongodb.rst

```diff
@@ -0,0 +1,751 @@
+-------------------------
+
+In order to create a new MongoDB similarity index, you need to specify either
+the `embeddings` or `model` argument to
+:meth:`compute_similarity() <fiftyone.brain.compute_similarity>`. Here's a few
+possibilities:
+
+.. code:: python
+    :linenos:
+
+    import fiftyone as fo
+    import fiftyone.brain as fob
+    import fiftyone.zoo as foz
+
+    dataset = foz.load_zoo_dataset("quickstart")
+    model_name = "clip-vit-base32-torch"
+    model = foz.load_zoo_model(model_name)
+    brain_key = "mongodb_index"
+
+    # Option 1: Compute embeddings on the fly from model name
+    fob.compute_similarity(
+        dataset,
+        model=model_name,
+        embeddings="embeddings",  # the field in which to store the embeddings
+        backend="mongodb",
+        brain_key=brain_key,
+    )
+
+    # Option 2: Compute embeddings on the fly from model instance
+    fob.compute_similarity(
+        dataset,
+        model=model,
+        embeddings="embeddings",  # the field in which to store the embeddings
+        backend="mongodb",
+        brain_key=brain_key,
+    )
+
+    # Option 3: Pass precomputed embeddings as a numpy array
+    embeddings = dataset.compute_embeddings(model)
+    fob.compute_similarity(
+        dataset,
+        embeddings=embeddings,
+        embeddings_field="embeddings",  # the field in which to store the embeddings
+        backend="mongodb",
+        brain_key=brain_key,
+    )
+
+    # Option 4: Pass precomputed embeddings by field name
+    # Note that MongoDB vector indexes require list fields
+    embeddings = dataset.compute_embeddings(model)
+    dataset.set_values("embeddings", embeddings.tolist())
+    fob.compute_similarity(
+        dataset,
+        embeddings="embeddings",  # the field that contains the embeddings
+        backend="mongodb",
+        brain_key=brain_key,
+    )
+
+.. note::
+
+    You can customize the MongoDB index by passing any
+    :ref:`supported parameters <mongodb-config-parameters>` as extra kwargs.
+
+.. _mongodb-patch-similarity-index:
+
+Create a patch similarity index
+-------------------------------
+
+.. warning::
+
+    The MongoDB backend does not yet support indexing object patches, so the
+    code below will not yet run. Check back soon!
+
+You can also create a similarity index for
+:ref:`object patches <brain-object-similarity>` within your dataset by
+including the `patches_field` argument to
+:meth:`compute_similarity() <fiftyone.brain.compute_similarity>`:
+
+.. code:: python
+    :linenos:
+
+    import fiftyone as fo
+    import fiftyone.brain as fob
+    import fiftyone.zoo as foz
+
+    dataset = foz.load_zoo_dataset("quickstart")
+
+    fob.compute_similarity(
+        dataset, 
+        patches_field="ground_truth",
+        model="clip-vit-base32-torch",
+        embeddings="embeddings",  # the attribute in which to store the embeddings
+        backend="mongodb",
+        brain_key="mongodb_patches",
+    )
+
+.. note::
```

Comment:
This Note dups the note immediately above the `Create a patch similarity index` example

Reply:
does it? this one is about extra kwargs, while the one above is basically saying "umm, yeah this section doesn't actually work yet!"

Reply:
All good, by the "one above" I meant the note above the "doesn't work yet". Nbd v minor